cosmo_tester/framework/test_hosts.py

```diff
@@ -600,7 +580,10 @@ def _old_wait_for_manager(self):
                        service in ['postgresql', 'rabbitmq']):
                     if instance['state'] != 'running':
                         raise StandardError(
```

Comment:
could well be a timeouterror :P

Reply:
Not until the test framework definitely works on py3, haven't got to that yet.